pallets/block-rewards/src/lib.rs

```diff
@@ -131,7 +131,8 @@ pub mod pallet {
 			+ MaxEncodedLen
 			+ FixedPointOperand
 			+ Into<<<Self as Config>::Currency as CurrencyT<Self::AccountId>>::Balance>
-			+ MaybeSerializeDeserialize;
+			+ MaybeSerializeDeserialize
+			+ From<cfg_primitives::Balance>;
```

Comment:
If this line is added because of `DEFAULT_COLLATOR_STAKE` type, I'll add this value as a constant too.

I'm thinking about what would happen if another chain wanted to use `block-rewards` as a dependency. It could be really cool to offer this crate in a more standalone way (it is already almost in this way).

I only wanted to leave this idea here. I'm ok as it is now 🚀

Reply:
Think this is a good idea.

Reply:
Definitely agree. Refactored this together with the collator group id to make the pallet generic enough to be used by others: [02060eb](https://github.com/centrifuge/centrifuge-chain/pull/1198/commits/02060ebdecb65d1eadacb8b62bfd441d4f54a984)